indexer/services/comlink/src/request-helpers/error-handler.ts

```
@@ -17,7 +17,7 @@ export function handleUnexpectedFieldErrors(
     forEach(object, (value, key) => {
       if (!isEqual(value, base[key])) {
         if (isObject(value) && isObject(base[key])) {
-          difference(value, base[key], `${parentPath}${key}.`);
+          difference(value, base[key] as Record<string, {}>, `${parentPath}${key}.`);
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)